src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+                                                    '\n with Caitlyn McCloskey from PEG as moderator']}/>
+                </EventWithDate>
+
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">Energy & Transportation</Subtitle>
+                    <Text fontSize="0.2rem">Co-sponsored by the
+                    <Link href="http://pennenvironmentalgroup.weebly.com/"> Penn Environmental Group</Link></Text>                
+                    <EventWithDateInfo date="APR 1 5 PM EST"
+                                    description={['Dr. Arthur van Benthem @' ,'Wharton http://www.arthurvanbenthem.com/', 
+                                                    '\n Dr. Peter Psarras @', 'Penn Carbon Capture https://kleinmanenergy.upenn.edu/news-insights/penn-carbon-capture-expert-joins-biden-administration/', 
+                                                    '\n Ally Warson @', 'UP Partners https://www.up.partners/ally-warson', 
+                                                    '\n E. Mitchell Swaan @', 'PEA https://philaenergy.org/about-pea/board-and-staff/e-mitchell-swann-secretary/',
+                                                    '\n with Ben May from', 'ThinkOcean https://thinkoceanglobal.org/ben-may' ,'as moderator']}/>
+                </EventWithDate>
+
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">Food & Agriculture</Subtitle>
+                    <Text fontSize="0.2rem">Co-sponsored by the
+                    <Link href="https://www.agribusiness-club.com/"> Wharton AgriBusiness Club</Link></Text>                
+                    <EventWithDateInfo date="APR 2 4 PM EST"
+                                    description={['Dr. Zhengxia Dou @' ,'Penn Vet https://www.vet.upenn.edu/research/centers-laboratories/research-laboratory/research-laboratory/dou-laboratory', 
+                                                    '\n Dr. David Galligan @', 'Penn Vet https://www.vet.upenn.edu/people/faculty-clinician-search/davidgalligan', 
+                                                    '\n Alex Behar @', 'Cultivian Sandbox https://cultiviansbx.com/news-and-events/about/meet-the-team/alex-behar/', 
+                                                    '\n with Akaash Padmanabha from Penn Dining Advisory as moderator']}/>
+                </EventWithDate>
+                <br/>
+                <Button href="https://tinyurl.com/PCVPrizeS21">Get the Link</Button>
+            </EventGroup>
+
+            <EventGroup>
+                <Subtitle fontColor={GREEN}>PCV Prize Launch Event</Subtitle>
+                <Text>You will be briefed on prize details, hear from Dr. Majumdar, and get a chance to network with peers.</Text>
+                <EventWithDate>
+                    <Subtitle fontSize="0.6rem">Opening Keynote with Dr. Arun Majumdar</Subtitle>
```

Comment:
If every Subtitle has the same fontSize, might be worthwhile looking into adding a prop to Subtitle to indicate that you want this fontSize instead of manually overriding EVERY Subtitle. 

DRY principle ; )